packages/docusaurus-theme-classic/src/theme/Navbar/index.js

```diff
@@ -17,6 +17,8 @@ import SearchBar from '@theme/SearchBar';
 
 import classnames from 'classnames';
 
+import useTheme from '../../hooks/theme';
```

Comment:
should be @theme/hooks/theme

Reply:
this means I should put hooks under `theme` directory is it?

Reply:
```suggestion
import useTheme from '@theme/hooks/theme';
```

iirc this works as well

Reply:
Moved hooks into `theme/`, not sure if this is intended or not?